### ISSUE TITLE
New accounts do not accept email

### ISSUE THREAD

**Issue by oganm** [bgreenwell/RBitmoji#6]
New bitmoji accounts can only be opened through snapchat it seems and accounts opened with snapchat cannot be logged in via email. Any way to get ids of accounts opened through snapchat?

**Comment by bgreenwell** [comment]
@oganm Correct, it should be the email associated with your Snapchat. You can see which email is listed in your Snapchat settings. Please let me know if it still does not work. Same goes for your password as well.

**Comment by bgreenwell** [comment]
@oganm If that does not work, I believe you can also sign up for Bitmoji with an email through the iOS or Android Bitmoji app. That account can then also be linked to your Snapchat (if you have one); I believe that is what I did.

**Comment by bgreenwell** [comment]
Just started another Bitmoji account through the iOS app with a new email/password. Works like a charm. I'll see if I can work on a fix for individuals who only want to use their Snapchat login---no promises though!

![image](https://user-images.githubusercontent.com/3777755/46472763-373f5580-c7ac-11e8-9a8b-8557e74ed3ab.png)

**Comment by oganm** [comment]
Hmm it seems like that is iOS specific. At least I am yet to find a way to do that in android app or their web page.

Using snapchat associated apps doesn't seem to work. I tried to cheat the system a little and ask to reset my password even though I had none, and it doesn't let you do it. I have tried your function with my snapchat email/password but alas that also fails.

**Comment by bgreenwell** [comment]
Does the Android app give you the option to log out? If so, do that and you see "Log in with Snapchat" and "Sign up with Email". I'd be surprised if the Android app did not offer the ability to sign up with email (after logging out) like the iOS app.

**Comment by oganm** [comment]
I am also surprised that they don't. Especially through their actual website. I guess they are trying to phase it out?

There is a "log in with Email" option as pictured below

![screenshot_20181005-133853](https://user-images.githubusercontent.com/6352379/46559109-1eb26680-c8a4-11e8-9f02-e6914dfc67cc.jpg)

And it only allows logging in with existing accounts

![screenshot_20181005-134013](https://user-images.githubusercontent.com/6352379/46559155-3db0f880-c8a4-11e8-8957-b19b8f8e2074.jpg)

No sign up option. Forget your password tells you that you email is invalid if you try to use a snapchat password.

Same thing with the website

![image](https://user-images.githubusercontent.com/6352379/46559217-8ec0ec80-c8a4-11e8-8f34-300c6d824c5d.png)

**Comment by bgreenwell** [comment]
This saddens me 😭. I'm going to leave this issue open in the case that someone finds/has a solution!

**Comment by oganm** [comment]
I will look around for workaroounds when I have time. Worst case is an ios emulator

**Comment by oganm** [comment]
I have found a way to access your id fairly easily even if it is a snapchat associated account. If you have the bitmoji chrome extension, copying image address of any bitmoji gives you a link like this:

```
https://render.bitstrips.com/v2/cpanel/ff875ee5-3acd-4edc-b200-e80a748277b3-b1962238-3114-4de5-8ab4-89b5d604e09a-v1.png?transparent=1&palette=1
```

The last 5 `-` separated strings before the v1 gives you your id (including the v1 itself). So mine is `'b1962238-3114-4de5-8ab4-89b5d604e09a-v1'`

**Comment by bgreenwell** [comment]
This is initially what I had to do to get my ID!!
